practice_05.25.19_interface/Task2_BankAccount/BankAccount.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace practice_05._25._19_interface.Task2_BankAccount
+{
+    public class BankAccount
+    {
+        protected decimal currentBalance;
+
+        public string AccountOwner { get; }
+
+        public BankAccount(decimal currentBalance, string balanceOwner)
+        {
+            this.currentBalance = currentBalance;
+            this.AccountOwner = balanceOwner;
+        }
+
+        public decimal CurrentBalance
+        {
+            get
+            {
+                return currentBalance;
+            }
+            set
+            {
+                if (value < 0)
+                    throw new ArgumentException();
+                else
+                    currentBalance = value;
+            }
+        }
+
+        public decimal CloseAccount()
+        {
+            decimal returnedBalance = this.currentBalance;
+            this.currentBalance = 0;
+
+            return returnedBalance;
+        }
+
+        public virtual void ShowAccountInformation()
```

Comment:
It is enough to override ToString() in all these classes.
No need to create a new method with the same logic.

Reply:
fixed - ToString method was overrided